src/main/java/me/kixstar/dungeons/dungeons/ChestUtils.java

```diff
@@ -10,14 +12,24 @@
 public class ChestUtils {
 
     private static final Random random = new Random();
-
     /**
      * distributeItems distributes a list of items into different chests, randomly
      * @param chests
      * @param items
      */
     static void distributeItems(List<Chest> chests, List<ItemStack> items) {
-
+        if (chests != null && items != null && !(items.isEmpty())) {
+            for (Chest chest : chests) {
+                List<ItemStack> itemStacks = null;
```

Comment:
NullPointerException

You're not actually assigning anything to the list of ItemStacks, so you can't add/remove/manipulate it later on, because it doesn't exisit